src/styles/levelSelect.scss

```diff
@@ -64,3 +64,28 @@
   justify-self: left;
   padding-bottom: 2.5px;
 }
+
```

Comment:
Font size seems a little too big, Figma has it set at 18, let’s try that and see how it looks. Also, let’s add an opacity to .dropdown-root, the Figma uses 70%.